### PR TITLE
Notification for resolved errors

### LINKED ISSUE #25: Notification of when errors resolve
After #20 is done.

### PR DESCRIPTION
**Pull request by mboneil10**
Closes #25

### CODE REVIEW

File: lib/bojangles.rb [hunk 1]
```diff
@@ -95,18 +95,28 @@ def go!
...
+  def message_html(error_messages, current)
     message = ["This message brought to you by Bojangles, UMass Transit's monitoring service for the PVTA realtime bus departures feed."]
-    message << message_list(error_messages)
+    message << message_list(error_messages, current)
     message.flatten.join '<br>'
   end
 
-  def message_list(error_messages)
-    heading = 'Bojangles has noticed the following errors:'
+  def message_list(error_messages, current)
```
Comment (dfaulken) [inline lib/bojangles.rb]: Boolean arguments are generally an antipattern, that is, it's difficult to tell from looking at them what they're for.

I'd use a kwarg here.

File: lib/bojangles.rb [hunk 2]
```diff
@@ -95,18 +95,28 @@ def go!
...
+    if current == true
```
Comment (dfaulken) [inline lib/bojangles.rb]: For checking the value of a boolean, anyhow, you don't need `== true`.

File: lib/bojangles.rb (hunk 1 above)
Comment (dfaulken) [reply lib/bojangles.rb]: For example

```ruby
message_list(resolved_error_messages, current: true)
```

And

```ruby
def message_list(error_messages, current:)
```

which will require the kwarg.

**Review by dfaulken — CHANGES_REQUESTED** [review]
Small changes.

File: lib/bojangles.rb (hunk 1 above)
Comment (Anbranin) [reply lib/bojangles.rb]: wow I did not know what a "quarg" was for a while. I'm like what sorcery is in Ruby > 2 that I didn't know about

Comment (werebus) [reply lib/bojangles.rb]: quarg-cheese

Comment (dfaulken) [reply lib/bojangles.rb]: Yep that's it

File: lib/bojangles.rb (hunk 2 above)
Comment (mboneil10) [reply lib/bojangles.rb]: Don't know what I was thinking.

**Review by Anbranin — APPROVED** [review]

**Review by dfaulken — APPROVED** [review]

**Pull request merged**